UPGRADING.md

```diff
@@ -130,5 +116,23 @@ mistakes.
 
 ### Modules
 
+#### `x/params`
+
 * The `x/param` module has been depreacted in favour of each module housing and providing way to modify their parameters. Each module that has parameters that are changable during runtime have an authority, the authority can be a module or user account. The Cosmos-SDK team recommends migrating modules away from using the param module. An example of how this could look like can be found [here](https://github.com/cosmos/cosmos-sdk/pull/12363). 
-    * The Param module will be maintained until April 18, 2022. At this point the module will reach end of life and be removed from the Cosmos SDK.
+* The Param module will be maintained until April 18, 2023. At this point the module will reach end of life and be removed from the Cosmos SDK.
+
+#### `x/gov`
+
+The `gov` module has been greatly improved. The previous API has been moved to `v1beta1` while the new implementation is called `v1`.
+
+In order to submit a proposal with `submit-proposal` you now need to pass a `proposal.json` file.
+You can still use the old way by using `submit-legacy-proposal`. This is not recommended.
+More information can be found in the gov module [client documentation](https://docs.cosmos.network/v0.46/modules/gov/07_client.html).
+
+### Protobuf
+
+The `third_party/proto` folder that existed in [previous version](https://github.com/cosmos/cosmos-sdk/tree/v0.45.3/third_party/proto) now does not contains directly the [proto files](https://github.com/cosmos/cosmos-sdk/tree/release/v0.46.x/third_party/proto).
```

Comment:
Why do we need this folder? IMO we can completely delete the third_party folder in v0.46+

Reply:
Ah no, it's because we do a custom generation of google.Any in `protocgen-any.sh`, so we need the any.proto file.